doc/nrf/releases_and_maturity/releases/release-notes-3.1.1.rst

```diff
@@ -0,0 +1,198 @@
+
+.. _ncs_release_notes_311_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE, OS, and tool support
+=========================
+
+* Added macOS 26 support (Tier 3) to the table listing :ref:`supported operating systems for proprietary tools <additional_nordic_sw_tools_os_support>`.
+* Updated the required `SEGGER J-Link`_ version to v8.60.
+
+Developing with PMICs
+=====================
+
+* Added the :ref:`ug_npm1304_developing` documentation.
+
+Developing with coprocessors
+============================
+
+* Added support for the nRF54LM20 FLPR.
+
+Security
+========
+
+* Added CRACEN and nrf_oberon driver support for nRF54LM20.
+  For the list of supported features and limitations, see the :ref:`ug_crypto_supported_features` page.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Matter
+------
+
+* Added support for the :zephyr:board:`nrf54lm20dk` board.
+* Included fixes for the following certification test cases:
+
+  * TC-IDM-10.3
+  * TC-OPCREDS-3.8
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+nRF Desktop
+-----------
+
+* Added application configurations for the :zephyr:board:`nrf54lm20dk` board.
+  The configurations are supported through the ``nrf54lm20dk/nrf54lm20a/cpuapp`` board target.
+  For details, see the :ref:`nrf_desktop_board_configuration` page.
+  The :zephyr:board:`nrf54lm20dk` board support is experimental.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Bluetooth samples
+-----------------
+
+* Added experimental support for the :zephyr:board:`nrf54lm20dk` board in the following samples:
+
+  * :ref:`bluetooth_central_hids`
+  * :ref:`peripheral_hids_keyboard`
+  * :ref:`peripheral_hids_mouse`
+
+Bluetooth Fast Pair samples
+---------------------------
+
+* Added experimental support for the :zephyr:board:`nrf54lm20dk` board in all Fast Pair samples.
+
+Matter samples
+--------------
+
+* Added support for the :zephyr:board:`nrf54lm20dk` board in all Matter samples.
+
+PMIC samples
+------------
+
+* Updated:
+
+  * By renaming the nPM1300: Fuel Gauge sample to :ref:`npm13xx_fuel_gauge`.
+    The wiring is changed for all targets.
+    Refer to the :ref:`Wiring table <npm13xx_fuel_gauge_wiring>` in the sample documentation for details.
+  * By renaming the nPM1300: One button sample to :ref:`npm13xx_one_button`.
+    The wiring is changed for all targets.
+    Refer to the :ref:`Wiring table <npm13xx_one_button_wiring>` in the sample documentation for details.
+
+Thread samples
+--------------
+
+* Added support for the :zephyr:board:`nrf54lm20dk` board in the :ref:`ot_cli_sample` and :ref:`ot_coprocessor_sample` samples.
+
+Other samples
+-------------
+
+* :ref:`coremark_sample` sample:
+
+  * Added support for the :zephyr:board:`nrf54lm20dk` board target.
```

Comment:
Shouldn't this be just board (not board target)?

Reply:
@alstrzebonski : Could you confirm?